engine/server/engine/main.go

```diff
@@ -42,6 +44,7 @@ const (
 	logMethodAlongWithLogLine = true
 	functionPathSeparator     = "."
 	emptyFunctionName         = ""
+	webappPortAddr            = ":9711"
```

Comment:
I think that this value should be put in a centralized place as we did for the `grpcPortNum`, so we can see on the same file that the port number `9710` is for GRPC and `9711` is taken for the HTTP server and if you have to change the number, so change it only in one place.